pyproject.toml

```diff
@@ -19,6 +19,8 @@ meinheld = "~1.0.2"
 Werkzeug = "~2.0.3"
 cloudpickle = "~2.1.0"
 crypto-cpp-py = "~1.0.4"
+marshmallow = "^3.17.0"
```

Comment:
Please redo the installation of these packages by doing the tilde (~) installation. It has to be done with `poetry add` rather than manually, otherwise the content-hash in poetry.lock doesn't get calculated properly which prevents later docker image building. Speaking of the content-hash, I see it's reported by GitHub as conflicting with master, you should probably rebase/merge.

Reply:
Should I keep these versions but with a tilde instead of caret symbol? Wouldn't the content-hash be changed then?
I did install these two packages using `poetry add`
It's the same situation as [here](https://stackoverflow.com/questions/62378386/how-can-i-add-a-dependency-in-poetry-using-tilde-instead-of-caret).

Reply:
Doing `poetry add` but with a tilde should work (and it would also correctly update the content-hash): so `poetry add marshmallow~3.17.0`